### PR TITLE
New package: ProcDump-for-Linux-1.0.1

### PR DESCRIPTION
**Pull request by ndowens**
This is procdump from Microsoft's sysinternals

### CODE REVIEW

**Comment by ndowens** [comment]
Just noticed the naming rules, will fix as upstream has it listed as ProcDump-for-Linux

**Comment by Johnnynator** [comment]
You should include this patch https://github.com/Microsoft/ProcDump-for-Linux/commit/4654429b1c4b21a630b2157dcb1a02628b3eb9d2 it should fix the issues you see for musl currently

File: srcpkgs/ProcDump-for-Linux/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+revision=1
+build_style=gnu-makefile
+makedepends="zlib-devel"
+depends="gdb"
+short_desc="Provides applications developers to create core dumps"
+maintainer="<ndowens@yahoo.com>"
+license="MIT"
+homepage="https://github.com/Microsoft/ProcDump-for-Linux"
+distfiles="https://github.com/Microsoft/${pkgname}/archive/${version}.tar.gz"
+checksum="284b92a83ea7763b3edb5a300b8ef97c2c2a4a69c2efaca4822423b71c605a14"
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: doesn't need to be quoted

**Comment by Johnnynator** [comment]
> Using that patch gives: https://pastebin.com/kU9t8SHg

Ok. You can set the package to broken for musl archs. Add smth like this after checksum.

```
case "${XBPS_TARGET_MACHINE}" in
        *-musl) broken="https://travis-ci.org/void-linux/void-packages/jobs/471949081";;
esac
```

File: srcpkgs/ProcDump-for-Linux/template (hunk 1 above)
Comment (ndowens) [reply srcpkgs/ProcDump-for-Linux/template]: Any of the template or just those parts?

**Comment by ndowens** [comment]
I don't know why x86_64 failed, builds and pkg'd fine on my x86_64 and previous commits

File: srcpkgs/ProcDump-for-Linux/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+makedepends=zlib-devel
+depends=gdb
+short_desc="Provides developers a convenient way to create core dumps"
+maintainer="<ndowens@yahoo.com>"
+license=MIT
+homepage=https://github.com/Microsoft/ProcDump-for-Linux
+distfiles=https://github.com/Microsoft/${pkgname}/archive/${version}.tar.gz
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: distfiles and homepage needs to be quoted

File: srcpkgs/ProcDump-for-Linux/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ProcDump-for-Linux'
+pkgname=ProcDump-for-Linux
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+makedepends=zlib-devel
+depends=gdb
+short_desc="Provides developers a convenient way to create core dumps"
+maintainer="<ndowens@yahoo.com>"
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: missing name

File: srcpkgs/ProcDump-for-Linux/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ProcDump-for-Linux'
+pkgname=ProcDump-for-Linux
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+makedepends=zlib-devel
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: make depends and depends need to be quoted

File: srcpkgs/ProcDump-for-Linux/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Provides developers a convenient way to create core dumps"
+maintainer="<ndowens@yahoo.com>"
+license=MIT
+homepage=https://github.com/Microsoft/ProcDump-for-Linux
+distfiles=https://github.com/Microsoft/${pkgname}/archive/${version}.tar.gz
+checksum=284b92a83ea7763b3edb5a300b8ef97c2c2a4a69c2efaca4822423b71c605a14
+
+case "${XBPS_TARGET_MACHINE}" in
+	*-musl) broken="https://travis-ci.org/void-linux/void-packages/jobs/471949081";;
+	esac
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: esac shouldn't be indented here

File: srcpkgs/ProcDump-for-Linux/template [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'ProcDump-for-Linux'
+pkgname=ProcDump-for-Linux
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+makedepends="zlib-devel"
+depends="gdb"
+short_desc="ProcDump-for-Linux provides a way to create core dumps"
+maintainer="<ndowens@yahoo.com>"
+license=MIT
```
Comment (maxice8) [inline srcpkgs/ProcDump-for-Linux/template]: license= field needs to be quoted

**Comment by ndowens** [comment]
Done

**Pull request merged**